blog/Konferencja Tłumaczy 2023 - relacja/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title:
+  'Konferencja Tłumaczy 2023 – relacja'
+authors: jkula
+date: '2023-10-19'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'konferencje'
+  - 'społecznosc'
+coverImage: 'KT2023-logo.jpg'
+---
+
+Konferencja Tłumaczy 2023 już za nami. Ale poniżej czeka na Was relacja. Zapraszamy do lektury!
+
+<!--truncate-->
+
+## W dniu św. Hieronima, patrona tłumaczy
+
+Co roku, 30 września, w dniu św. Hieronima, tłumacza Biblii, uznawanego za patrona tłumaczy, obchodzony jest Międzynarodowy Dzień Tłumacza. I w tym samym terminie, już od kilku lat, organizowana jest Konferencja Tłumaczy. Nie inaczej było i w tym roku.
```

Comment:
W konfiguracji repo ustawiłem, żeby długość linii była max 80 znaków. Jeśli uda Ci się odpalić Prettier w VS Code, to Ctrl+S zrobi to automatycznie. TO jest pod koniec mojego guide'a w README. Daj znać, jeśli coś Ci nie działa.